### PR TITLE
sc-2918 Payouts #get and #list

### PR DESCRIPTION
**Pull request by jmatheus**
feat: list, get payouts

Type of Change
--------------

* [ ] Bug fix (non-breaking change which fixes an issue)
* [x] New feature (non-breaking change which adds functionality)
* [ ] Breaking change (would cause existing functionality to not work as expected)
* [ ] This change requires a documentation update
* [ ] This change is in scope for PCI
* [ ] This requires approval from UX
* [ ] This requires approval from Marketing

Links
-----

Steps for Testing/QA
--------------------

### CODE REVIEW

File: spec/stubs/payouts.rb [hunk 1]
```diff
@@ -0,0 +1,100 @@
+module Stubs
+  class Payout
+    class << self
+      def success_get(payout_id)
+        response_body = {
+          "id": 1,
+          "type": "payout",
+          "data": {
+            "id": "po_xyz",
+            "account_id": "449e7a5c-69d3-4b8a-aaaf-5c9b713ebc65",
```
Comment (jnwaletzko) [inline spec/stubs/payouts.rb:10]: Are we surfacing the wrong id here? This should be a public id

**Review by jnwaletzko — COMMENTED** [review]
One question otherwise looks great

File: spec/stubs/payouts.rb (hunk 1 above)
Comment (jmatheus) [reply spec/stubs/payouts.rb]: Oh we are, but only in our docs. Just tested the API and we're surfacing the public id.
I fixed it here and we can change our docs in a follow-up

**Review by jnwaletzko — APPROVED** [review]
perfect, thanks for checking that 👍

**Pull request merged**